src/main/resources/plugin.yml

```diff
@@ -5,7 +5,7 @@ description: ${project.description}
 website: ${project.organization.url}
 author: ${project.organization.name}
 authors: [ Wolf2323, SaltyAimbOtter, seyfahni, joblo2213, bundabrg, Co0sh ]
-api-version: 1.17
+api-version: 1.18
```

Comment:
I want to move this value as a variable to the pom.xml file. I would do this in this PR

Reply:
This has been added.